storage/store.go

```diff
@@ -0,0 +1,38 @@
+package storage
+
+import (
+	"context"
+	"sync"
+)
+
+type Store struct {
+	mu sync.RWMutex
+	// highest block height
+	blockHeight uint64
+}
+
+// NewStore returns a new in-memory Store implementation.
+func NewStore() *Store {
+	return &Store{
+		mu: sync.RWMutex{},
```

Comment:
nit: you don't need to initialize `sync.RWMutex` since the empty value is the same as `sync.RWMutex{}`

Reply:
Good point :+1: 
Updated in https://github.com/onflow/flow-evm-gateway/pull/11/commits/78b1e3948d8608573a2b6dcd7d987f1f13cc5d31